src/components/Profile/ProfilePage1/index.tsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable import/prefer-default-export */
+import React from 'react';
+import { View, StatusBar } from 'react-native';
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';
+import styles from './styles';
+import ProfileHeader from '../ProfileHeader';
+import ProfileContainer from '../ProfileContainer';
+import HomeContainer from '../HomeContainer';
+
+export function ProfilePage1() {
```

Comment:
NIT: can this just be Profile Page if there are no other components called Profile Page?

Reply:
Changed this to ProfilePage